lib/smart_proxy_dynflow/io_buffer.rb

```diff
@@ -88,7 +88,7 @@ def write_available!
           n = @io.write_nonblock(@buffer)
           @buffer = @buffer[n..-1]
         end
-      rescue IO::WaitWritable # rubocop:disable Lint/HandleExceptions
+      rescue IO::WaitWritable
```

Comment:
```suggestion
      rescue IO::WaitWritable # rubocop:disable Lint/SupressedException
```

Reply:
Updated this changes here https://github.com/theforeman/smart_proxy_dynflow/pull/118